Cargo.toml

```diff
@@ -19,10 +19,10 @@ thiserror = "1.0.37"
 serde_json = "1.0.85"
 lazy_static = "1.4.0"
 
-near-crypto = ">0.22,<0.24"
-near-primitives = ">0.22,<0.24"
-near-chain-configs = ">0.22,<0.24"
-near-jsonrpc-primitives = ">0.22,<0.24"
+near-crypto = ">0.22,<0.25"
+near-primitives = ">0.22,<0.25"
+near-chain-configs = ">0.22,<0.25"
+near-jsonrpc-primitives = ">0.22,<0.25"
```

Comment:
I feel that the changes in the code (e.g. `TransactionV0`) mean that older versions of the crates won't work, will they?

Reply:
It compiles with older version (you can see that I haven't changed the lib itself)
We don't use Cargo.lock, that's why we would end up with latest 0.24 version, and it has some changes in API :)

Though, I'm not so sure about semver.

Reply:
Ah, those are examples. Sorry, it is fine indeed